src/Altinn.Platform/Altinn.Platform.Receipt/Tests/Health/HealthCheckTests.cs

```diff
@@ -0,0 +1,57 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Platform.Receipt;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Xunit;
+
+namespace Altinn.Platform.Receipt.UnitTest
+{
+    /// <summary>
+    /// Health check 
+    /// </summary>
+    public class HealthCheckTests : IClassFixture<WebApplicationFactory<Startup>>
+    {
+        private readonly WebApplicationFactory<Startup> _factory;
+
+        /// <summary>
+        /// Default constructor
+        /// </summary>
+        /// <param name="factory">The web applicaiton factory</param>
+        public HealthCheckTests(WebApplicationFactory<Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        /// <summary>
+        /// Verify that component responds on health check
+        /// </summary>
+        /// <returns></returns>
+        [Fact]
+        public async Task VerifyHeltCheck_OK()
+        {
+            HttpClient client = GetTestClient();
+
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, "/health")
+            {
+            };
+
+            HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+            string content = await response.Content.ReadAsStringAsync();
+
```

Comment:
this blank line reveals that we might be missing stylecop in this project?

Reply:
yes, it is not in all projects